tools/bin/ecflow/processes/main/meteorological/get_metdata.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+"""
+get_metdata.py 
+usage: <python> <get_metdata.py> <configuration.cfg>
+This script downloads meteorological data through xarray from
+http://thredds.northwestknowledge.net:8080/thredds/reacch_climate_MET_catalog.html
+delivered through OPeNDAP. Because attributes are lost during download, 
+they are added back in. 
+"""
+import sys
+import numpy as np
+import xarray as xr
+import os
+import time
+import argparse
+from collections import OrderedDict
+from datetime import datetime, timedelta
+from tonic.io import read_config
+from monitor import model_tools
+######### ----------------------------------------###########
+
+#read in configuration file
+parser = argparse.ArgumentParser(description='Reorder dimensions')
+parser.add_argument('config_file', metavar='config_file', type=argparse.FileType('r'), 
+	nargs=1, help='configuration file')
+args = parser.parse_args()
+config_dict = read_config(args.config_file[0].name)
+
+#read in meterological data location
+met_loc = config_dict['ECFLOW']['Met_Loc']
+old_config_file = config_dict['ECFLOW']['old_Config'] 
+new_config_file = config_dict['ECFLOW']['new_Config']
+N = config_dict['ECFLOW']['Met_Delay']
+
+N = int(N)
+
+#get date and number of days
+date = datetime.now() - timedelta(days=N)
+num_day = date.timetuple().tm_yday - 1 #python 0 start correction
+year = date.strftime('%Y')
+date_format = date.strftime('%Y-%m-%d')
+
+#replace start date, end date and year in the configuation file
```

Comment:
did you intend to replace year? should `'MODEL_START_DATE'` be the same as `'MODEL_END_DATE'`